tests/behat/behat-features/src/PushingContext.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+/**
+ * @file
+ * The behat context for replication tests.
+ */
+
+use Behat\Mink\Exception\ExpectationException;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use GuzzleHttp\Client;
+
+/**
+ * Defines features from the pushing context.
+ */
+class PushingContext extends RawDrupalContext {
```

Comment:
Minor: This contains many general purpose helpers, so "pushing" is not a good category for them. I'd rather just call it ContentPoolContext